### ISSUE TITLE
Bug in local_independencies method of Bayesian Model

### ISSUE THREAD

**Issue by yashu-seth** [pgmpy/pgmpy#529]
``` python
>>> from pgmpy.models import BayesianModel
>>> model = BayesianModel()
>>> model.add_edges_from([('a', 'b'), ('a', 'c'), ('b', 'd')])
>>> model.local_independencies('b')
(b _|_ c | a)
>>> model.local_independencies('a')
Traceback (most recent call last):
  File "<pyshell#9>", line 1, in <module>
    model.local_independencies('a')
  File "C:\Python34\lib\site-packages\pgmpy-0.1.0-py3.4.egg\pgmpy\models\BayesianModel.py", line 378, in local_independencies
    set(self.get_parents(variable))])
  File "C:\Python34\lib\site-packages\pgmpy-0.1.0-py3.4.egg\pgmpy\independencies\Independencies.py", line 86, in add_assertions
    self.independencies.append(IndependenceAssertion(assertion[0], assertion[1], assertion[2]))
  File "C:\Python34\lib\site-packages\pgmpy-0.1.0-py3.4.egg\pgmpy\independencies\Independencies.py", line 171, in __init__
    raise exceptions.RequiredError('event2 needed')
pgmpy.exceptions.Exceptions.RequiredError: 'event2 needed'
```

**Comment by yashu-seth** [comment]
The problem is in finding the local independencies of the root node when there is only one root node.

**Comment by kshitij10496** [comment]
+1.
Even I have encountered this bug while working.
I think it would be better to raise a relevant exception or a `None` independencies object

**Comment by yashu-seth** [comment]
@kshitij10496  Yes i also think it should return a None object.

**Comment by yashu-seth** [comment]
@kshitij10496 I saw that you are already working on get_independencies. If you like please look into this too. If you feel otherwise let me know i'll fix it.

**Comment by kshitij10496** [comment]
@yashu-seth 
Yeah
I will definitely look into this

**Comment by ankurankan** [comment]
@kshitij10496 @yashu-seth Is this fixed ?

**Comment by yashu-seth** [comment]
@kshitij10496 please tell us about the status of this issue. If you are not free currently let me know i'll fix it.

**Comment by ankurankan** [comment]
Hello,

I am working on this.
I will send the PR by tomorrow

**Comment by yashu-seth** [comment]
PR #545

**Comment by khalibartan** [comment]
@ankurankan What you think we should return a `None` object or raise a relevant exception.

**Comment by chrisittner** [comment]
See PR #629, this can be closed i think.

**Comment by ankurankan** [comment]
Fixed in #629
